Library/Homebrew/formula_auditor.rb

```diff
@@ -339,6 +339,18 @@ def audit_postgresql
       end
     end
 
+    def audit_glibc
+      return if formula.name != "glibc"
+      return unless @core_tap
+
+      version = formula.version.to_s
+      return if version == "2.23"
+
+      problem "The glibc version can only be #{version} as this is the version used by our CI on Linux. " \
+              "Glibc is only there for users that have a system glibc with a lower version, " \
```

Comment:
Would it be sensible to use https://github.com/Homebrew/brew/blob/787e6cb75f8bfe403ef0a7fc8a4ed47a615bbcde/Library/Homebrew/brew.sh#L426 at all?

Reply:
Minimum version is 2.13, but we ship 2.23.

2.13 is the version we use to compile patchelf and binutils, and then you can compile all the formulae from source from there.

2.23 is the version you need if you want to use bottles but you have an older glibc on your system.

So not really the same. We could make them the same, but then we would drop support for a bunch of systems.